kv/scrapers_test.go

```diff
@@ -38,11 +38,11 @@ func initScraperTargetStoreService(s kv.SchemaStore, f influxdbtesting.TargetFie
 		}
 	}
 
-	for _, m := range f.UserResourceMappings {
-		if err := svc.CreateUserResourceMapping(ctx, m); err != nil {
-			t.Fatalf("failed to populate user resource mapping")
-		}
-	}
+	// for _, m := range f.UserResourceMappings {
```

Comment:
did you want to remove this or leave it commented?

Reply:
oo ill remove that thanks